pkg/model/api/config/authorization.go

```diff
@@ -92,16 +92,19 @@ func (n *NativeAuthorizationProvider) Verify() error {
 }
 
 type AuthorizationProvider struct {
-	Kubernetes KubernetesProvider          `json:"kubernetes,omitzero" yaml:"kubernetes,omitempty"`
-	Native     NativeAuthorizationProvider `json:"native,omitzero" yaml:"native,omitempty"`
+	// +optional
+	Kubernetes KubernetesProvider `json:"kubernetes,omitzero" yaml:"kubernetes,omitempty"`
+	// +optional
+	Native NativeAuthorizationProvider `json:"native,omitzero" yaml:"native,omitempty"`
```

Comment:
Same comment for the native word confusion. 
Here native doesn't have the same meaning in authn and authz